react/Button/Button.less

```diff
@@ -104,9 +104,11 @@
 }
 
 .root_isGhost {
-  border: 2px solid;
+  @border-thickness: 2px;
+
+  border: @border-thickness solid;
   font-weight: @sk-bold;
-  line-height: normal;
+  line-height: @touchableTextHeight - 2 * @border-thickness;
```

Comment:
For readability, could you add parentheses here? e.g. `touchableTextHeight - (@border-thickness * 2)`

Also, for consistency with CSS naming, could you also call this variable `@border-width`?